### ISSUE TITLE
Gulp fails when a flag is specified without tasks

### ISSUE THREAD

**Issue by richadr** [inikulin/gulp-ll#5]
If you run gulp without specifying the task but with a flag (any flag), it will fail.

`gulp --production` will fail
`gulp default --production` is OK

Here's a test to place in `test/test.js`:

```
test('Regression - Gulp fails when a flag is specified without tasks (GH-5)', function () {
    // NOTE: smoke test, this should pass without error
    return runGulp('--dev');
});
```

**Comment by inikulin** [comment]
Thank you for the report!

**Comment by xkr47** [comment]
@richadr Would you be interested in verifying my fix attempt? See #6 for info.

To test it, simply replace `"gulp-ll": "<version>"` in your package.json with `"gulp-ll": "xkr47/gulp-ll"` or if you need to access github over https then `"gulp-ll": "git+https://github.com/xkr47/gulp-ll"`
